pkg/services/ngalert/api/tooling/definitions/alertmanager.go

```diff
@@ -375,7 +375,7 @@ type GettableStatus struct {
 
 	// config
 	// Required: true
-	Config *PostableApiAlertingConfig `json:"config"`
+	Config *GettableApiAlertingConfig `json:"config"`
```

Comment:
@gotjosh as author of this code, do you remember why we use `PostableApiAlertingConfig`? I haven't found a good reason for this, and it seems to work as expected using `GettableApiAlertingConfig`. What am I missing?